lib/logstream/logstream_handlers.go

```diff
@@ -207,17 +199,31 @@ func (h *LokiLogHandler) GetLogLocation(consumers map[string]*ContainerLogConsum
 		sb.WriteString(fmt.Sprintf("&var-test=%s", testName))
 	}
 
-	var shorteningErr error
-	// try to shorten the URL only if we have all the required configuration parameters
-	if baseUrl != "" && dabshoardUrl != "" && h.loggingConfig.Grafana.BearerToken != nil {
-		var shortened string
-		shortened, shorteningErr = ShortenUrl(baseUrl, sb.String(), *h.loggingConfig.Grafana.BearerToken)
-		if shorteningErr == nil {
-			h.grafanaUrl = shortened
+	// Use short Grafana URL only in CI
+	if os.Getenv("CI") == "true" {
+		if h.loggingConfig.Grafana == nil || h.loggingConfig.Grafana.BaseUrlCI == nil {
+			return "", errors.New("grafana base URL for CI is not set in logging config")
+		}
+		baseUrl := *h.loggingConfig.Grafana.BaseUrlCI
+		baseUrl = strings.TrimSuffix(baseUrl, "/")
+		baseUrl = baseUrl + "/"
+
+		// try to shorten the URL only if we have all the required configuration parameters
+		shortUrl, err := ShortenUrl(baseUrl, sb.String(), *h.loggingConfig.Grafana.BearerToken)
+		if err != nil {
+			return "", err
+		}
+		h.grafanaUrl = shortUrl
+	} else {
+		if h.loggingConfig.Grafana == nil || h.loggingConfig.Grafana.BaseUrl == nil {
+			return "", errors.New("grafana base URL is not set in logging config")
```

Comment:
I don't think we need to return error. Why not display what we can? which is the part of url without hostname? in that way if you forget to set it in the config you will still have an usable url like: `/dashboard-ulr&var=1...` so you can just manually paste it after Grafana URL and get a working link